packages/sirius-web/backend/sirius-web/src/test/java/org/eclipse/sirius/web/application/controllers/objects/ObjectRestControllerIntegrationTests.java

```diff
@@ -0,0 +1,346 @@
+import org.springframework.transaction.annotation.Transactional;
+
+/**
+ * Integration tests of the object REST controller.
+ *
+ * @author arichard
+ */
+@Transactional
+@SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.RANDOM_PORT)
+public class ObjectRestControllerIntegrationTests extends AbstractIntegrationTests {
+
+
+    private static final String API_REST_PROJECTS = "/api/rest/projects";
+
+    private static final String API_REST_COMMITS = "/commits";
+
+    private static final String API_REST_ELEMENTS = "/elements";
+
+    private static final String API_REST_RELATIONSHIPS = "/relationships";
+
+    private static final String API_REST_ROOTS = "/roots";
+
+    private static final String ECORE_SAMPLE_PKG = """
+            {
+                "@id": "%s",
+                "@type": "EPackage",
+                "name": "Sample",
+                "nsURI": null,
+                "nsPrefix": null,
+                "eAnnotations": [],
+                "eClassifiers": [
+                    {
+                        "@id": "%s"
+                    }
+                ],
+                "eSubpackages": [],
+                "eSuperPackage": null
+            }
+            """.formatted(TestIdentifiers.EPACKAGE_OBJECT, TestIdentifiers.ECLASS_OBJECT);
+
+    private static final String ECORE_SAMPLE_ECLASS = """
+            {
+                "@id": "%s",
+                "@type": "EClass",
+                "name": "SampleEClass",
+                "instanceClassName": null,
+                "instanceClass": null,
+                "defaultValue": null,
+                "instanceTypeName": null,
+                "abstract": "false",
+                "interface": "false",
+                "eAnnotations": [],
+                "ePackage": { "@id": "%s" },
+                "eTypeParameters": [],
+                "eSuperTypes": [],
+                "eOperations": [],
+                "eAllAttributes": [],
+                "eAllReferences": [],
+                "eReferences": [],
+                "eAttributes": [],
+                "eAllContainments": [],
+                "eAllOperations": [],
+                "eAllStructuralFeatures": [],
+                "eAllSuperTypes": [],
+                "eIDAttribute": null,
+                "eStructuralFeatures": [],
+                "eGenericSuperTypes": [],
+                "eAllGenericSuperTypes": []
+            }
+            """.formatted(TestIdentifiers.ECLASS_OBJECT, TestIdentifiers.EPACKAGE_OBJECT);
+
+    @Autowired
+    private IGivenInitialServerState givenInitialServerState;
+
+    @LocalServerPort
+    private String port;
+
+    private String getHTTPBaseUrl() {
+        return "http://localhost:" + this.port;
+    }
+
+    @BeforeEach
+    public void beforeEach() {
+        this.givenInitialServerState.initialize();
+    }
+
+    @Test
+    @DisplayName("Test the GET '/projects/{projectId}/commits/{commitId}/elements' REST API, should return all elements of the given project")
```

Comment:
Please respect the naming convention of the tests and test methods, same comment for other tests methods.